integration-tests/http/__tests__/order/admin/rma-flows.spec.ts

```diff
@@ -0,0 +1,801 @@
+
+        fulfillableItem = orderResult.items.find(
+          (item) => item.detail.fulfilled_quantity < item.detail.quantity
+        )
+
+        // Ensure that there are no more fulfillable items
+        expect(fulfillableItem).toBeUndefined()
+
+        // Totals summary after fulfillment is done
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            difference_sum: 0,
+            refunded_total: 10,
+            transaction_total: 121,
+            pending_difference: 10,
+            current_order_total: 131,
+            // Note: When all items are fulfilled, the original_order_total goes to the new total
+            original_order_total: 131,
+            temporary_difference: 0,
+          })
+        )
+
+        /*
+            We can see that fulfillment affects the totals, so lets create and confirm 2 claims
+            and fulfill after
+          */
+
+        let claimWithInboundAndOutbound = (
+          await api.post(
+            "/admin/claims",
+            {
+              order_id: order.id,
+              type: ClaimType.REPLACE,
+              description: "Base claim",
+            },
+            adminHeaders
+          )
+        ).data.claim
+
+        orderResult = (await api.get(`/admin/orders/${order.id}`, adminHeaders))
+          .data.order
+
+        // Nothing changes from the previous expectation
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            difference_sum: 0,
+            refunded_total: 10,
+            transaction_total: 121,
+            pending_difference: 10,
+            current_order_total: 131,
+            original_order_total: 131,
+            temporary_difference: 0,
+          })
+        )
+
+        let inboundItem = orderResult.items[0]
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/inbound/items`,
+          { items: [{ id: inboundItem.id, quantity: 1 }] },
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/outbound/items`,
+          {
+            items: [
+              {
+                variant_id: product.variants[0].id,
+                quantity: 1,
+              },
+            ],
+          },
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/claims/${claimWithInboundAndOutbound.id}/request`,
+          {},
+          adminHeaders
+        )
+
+        orderResult = (await api.get(`/admin/orders/${order.id}`, adminHeaders))
+          .data.order
+
+        // Totals summary after all
+        expect(orderResult.summary).toEqual(
+          expect.objectContaining({
+            paid_total: 131,
+            difference_sum: -20,
+            refunded_total: 10,
+            transaction_total: 121,
+            // TODO: This looks wrong
+            pending_difference: -10,
```

Comment:
@carlos-r-l-rodrigues If a refund is made outside of payment collections created during an RMA flow, the pending difference is always accounting for that refund